api/Controllers/LocationController.cs

```diff
@@ -63,5 +63,18 @@ public async Task<ActionResult<List<Location>>> GetPCSSLocations()
 
             return Ok(locations);
         }
+
+        /// <summary>
+        /// Returns the list of locations and court rooms used in PCSS
+        /// </summary>
+        /// <returns>PCSS Locations and Court Rooms</returns>
+        [HttpGet]
+        [Route("pcss/court-rooms")]
+        public async Task<ActionResult<List<Location>>> GetPCSSLocationsAndCourtRooms()
+        {
+            var locations = await _locationService.GetPCSSLocationsAndCourtRooms();
+
+            return Ok(locations);
+        }
```

Comment:
An abstracted location lookup interface would be more desirable than a source specific interface.  One that is aware of the complete list of locations, rooms, and their codes as maintained by JUSTIN/CEIS, as well as PCSS's view and metadata associated to the same list.

The intent of the JASPER API is to abstract and decouple the source systems and data from the application.  Presenting the data JASPER needs to perform it's function as a unified view regardless of the source or sources of the data.